build.assets/version-ops/cmd/query-latest/main.go

```diff
@@ -0,0 +1,103 @@
+/*
+Copyright 2022 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Command query-latest returns the highest semver release for a versionSpec
+// query-latest ignores drafts and pre-releases.
+//
+// For example:
+//  query-latest -version-spec v8.1.5     -> v8.1.5
```

Comment:
We don't really need a flag here. This seems easier to me, what do you think?

```
$ query-latest v8.1.5
```

Reply:
Yeah -- the flag was pattern matching with what @russjones initially implemented in https://github.com/gravitational/teleport/pull/9236/files#diff-05bfff32689895234ec9c80d4b2e6bc10792758e23d3ae1f29f2ad3d0300c60fR31-R39

I'm ok leaving it as a flag for now -- it is verbose but largely moot.

Reply:
I looped back and made this change in 5cac414.